landcover.mss

```diff
@@ -766,7 +785,7 @@
       b/line-color: @tourism;
       b/line-opacity: 0.3;
       b/line-join: round;
-      b/line-cap: round;    
+      b/line-cap: round;
```

Comment:
whicespace noise

Reply:
This has already been removed as part of a whitespace cleanup.

Reply:
Yes, but it shouldn't be in *this* PR as well.